RoslynPluginGenerator/Interfaces/INuGetPackageHandler.cs

```diff
@@ -13,6 +13,9 @@ namespace SonarQube.Plugins.Roslyn
     /// </summary>
     public interface INuGetPackageHandler
     {
+
+        string localCacheRoot { get; }
```

Comment:
Responsibility for assigning this was moved into NuGetPackageHandler (with an optional parameter in its constructor). However, AnalyzerPluginGenerator requires this folder to perform additional assembly resolution so a property is exposed in the interface to allow this.